server/src/action/spotify/spotify.service.ts

```diff
@@ -0,0 +1,72 @@
+import { ForbiddenException, Injectable } from '@nestjs/common';
+import axios from 'axios';
+import { PrismaService } from '../../prisma/prisma.service';
+import { ServiceType } from '@prisma/client';
+import { EventEmitter2, OnEvent } from '@nestjs/event-emitter';
+import { ActionService } from '../action.service';
+import { SpotifyPlaybackStateDto } from './dto/spotify.dto';
+
+@Injectable()
+export class SpotifyService {
+    private readonly spotifyApiBaseUrl = 'https://api.spotify.com/v1';
+
+    constructor(
+        private readonly prisma: PrismaService,
+        private eventEmitter: EventEmitter2,
+        private actionService: ActionService,
+    ) {
+        this.eventEmitter.on('ExecuteSpotify.struct', (struct: SpotifyPlaybackStateDto) => {
+            struct.is_playing = false;
+        });
+    }
+
+    @OnEvent('ExecuteSpotify')
+    async getPlaybackState(structInfo: SpotifyPlaybackStateDto, actionId: number): Promise<any> {
+
+        const serviceId = await this.actionService.getServiceActions(actionId);
+
+
+        console.log("this is the service id", (await serviceId).id);
```

Comment:
remove all console.log on the file

Reply:
maybe also in [spotify.service.ts](https://github.com/AlexGuillard/Area/pull/157/files#diff-2ae31a822042e78c4e7d968fbf78fa06061b5622abf5387fac7c1d6405eec009)